app/objects/c_operation.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, name, agents, adversary, id=None, jitter='2/8', source=None,
         super().__init__()
         self.id = id
         self.start, self.finish = None, None
+        self.timeout = 180
```

Comment:
I think this should be a dynamic configuration option within the operations pane or global application configuration

Reply:
Renaming this to be more accurate.